frontend/app/views/spree/shared/_terms_and_conditions.en.html.erb

```diff
@@ -0,0 +1,64 @@
+
+Disclaimer as to ownership of trade marks, images of personalities and third party copyright
+Except where expressly stated to the contrary all persons (including their names and images), third party trade marks and content, services and/or locations featured on this Website are in no way associated, linked or affiliated with [Your Online Store URL] and you should not rely on the existence of such a connection or affiliation. Any trade marks/names featured on this Website are owned by the respective trade mark owners. Where a trade mark or brand name is referred to it is used solely to describe or identify the products and services and is in no way an assertion that such products or services are endorsed by or connected to [Your Online Store URL].
+
+Indemnity
+You agree to indemnify, defend and hold harmless [Your Online Store URL], its directors, officers, employees, consultants, agents, and affiliates, from any and all third party claims, liability, damages and/or costs (including, but not limited to, legal fees) arising from your use this Website or your breach of the Terms of Service.
+
+Variation
+[Your Online Store URL]  shall have the right in its absolute discretion at any time and without notice to amend, remove or vary the Services and/or any page of this Website.
+
+Invalidity
+If any part of the Terms of Service is unenforceable (including any provision in which we exclude our liability to you) the enforceability of any other part of the Terms of Service will not be affected all other clauses remaining in full force and effect. So far as possible where any clause/sub-clause or part of a clause/sub-clause can be severed to render the remaining part valid, the clause shall be interpreted accordingly. Alternatively, you agree that the clause shall be rectified and interpreted in such a way that closely resembles the original meaning of the clause /sub-clause as is permitted by law.
+
+Complaints
+We operate a complaints handling procedure which we will use to try to resolve disputes when they first arise, please let us know if you have any complaints or comments.
+
+Waiver
+If you breach these conditions and we take no action, we will still be entitled to use our rights and remedies in any other situation where you breach these conditions.
+
+Entire Agreement
+The above Terms of Service constitute the entire agreement of the parties and supersede any and all preceding and contemporaneous agreements between you and [Your Online Store URL]. Any waiver of any provision of the Terms of Service will be effective only if in writing and signed by a Director of [Your Online Store URL].
```

Comment:
Again, I'd prefer this to be in the `views/spree/checkout` folder instead of shared.